segmentation/utils/data_utils_test.py

```diff
@@ -13,7 +14,7 @@
 
 
 def _create_img_dir(temp_dir, fovs, imgs, img_sub_folder="TIFs"):
-    tif = np.random.randint(0, 100, 1024**2).reshape((1024, 1024)).astype('int8')
+    tif = np.random.randint(0, 100, 1024**2).reshape((1024, 1024)).astype("int8")
```

Comment:
You don't like single quotes I see :)

Reply:
I actually prefer single quotes to double quotes, but I saw that most of the quotes in the notebooks were double quotes, so I just turned all single quotes to double quotes for consistency. Let me know if you would like me to revert to the originally used quotes (a mix of single and double ones).